lib/testing/test_runner.sh

```diff
@@ -251,6 +251,28 @@ get_vlan_prefix()
     (IFS=:; set -- $prefix; printf "%s:%s:%s:%x::" "$1" "$2" "$3" $(($4 + $vid * 4096)))
 }
 
+check_python()
```

Comment:
Since a test function is run in a subshell, changed variables are not reflected in the parent shell. So I don't think it could easily be in the test file without checking Python every time.

Reply:
True, but does that matter? It's not like it's a huge overhead...